docs/configurations/application-config.mdx

```diff
@@ -45,6 +45,39 @@ There can only one instance of application configuration for the entire applicat
 }
 ```
 
+## Configuring widgets in FlexRow alignment
```

Comment:
🎉

Reply:
@SebaMutuku Nice to include this in the documentation. But I think the documentation here should be for `LIST` view widget which should be isolated from application configurations. I would suggest we create a separate directory for widget configurations then include this inside the `ListProperties` documentation.